box-ui/src/components/ts/SelectButton.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { FunctionComponent } from 'react';
+import { FormControl, InputLabel, Select, MenuItem } from '@mui/material';
+import { SelectButtonProps } from '../../utils/Props';
+
+const SelectButton: FunctionComponent<SelectButtonProps> = (props: SelectButtonProps) => {
```

Comment:
What is the use for props.value ? If it is meant to be the default value you should change the name